src/containers/loan/components/LoanProductDetails.js

```diff
@@ -15,9 +15,10 @@ export default function LoanProductDetails(props) {
             </Row>
             <Row>
                 <Col>
-                    Discount rate: <DiscountRateToolTip discountRate={prod.discountRate} />
+                    Interest p.a.: <DiscountRateToolTip discountRate={prod.discountRate} />
                 </Col>
-                <Col>{prod.discountRate * 100}%</Col>
+                <Col>{Math.floor((100 - (prod.discountRate * 100)) * 100) / 100}%</Col>
```

Comment:
same here as with locks: store the pa. interest in redux store (`modules/ethereum/lockTransactions.js parseLockProduct()` fx
keep the discountRate in store (as we use it on the form), just don't need to display it

Reply:
the correct formula (it's fine in the spreadsheet examples, just difficult to reverse engineer):
`1 / ( discountRate / PPM_DIV ) - 1)  / term  * 365`
it needs to be roundend to 4 decimals so when displaying it as `interestPa * 100` it will result to 14.15%